sdk/storage/azure-storage-blob/azure/storage/blob/aio/_download_async.py

```diff
@@ -356,6 +357,12 @@ async def _initial_request(self):
                     self.size = self._file_size
                 retry_active = False
 
+            except IncompleteReadError as error:
+                retry_total -= 1
+                if retry_total <= 0:
+                    raise ServiceResponseError(error, error=error)
+                await asyncio.sleep(1)
```

Comment:
Not a new issue, but why aren't we honoring the retry timeouts?

Reply:
I'm resolving this discussion, as the scope of Steven was really just the exception, but this is a good question.